### ISSUE TITLE
Pretty printing?

### ISSUE THREAD

**Issue by bburdette** [rotty/lexpr-rs#58]
I'd like to use s-expressions for a config file rather than json, but the single line output is pretty hard to read.  I'm not seeing an obvious pretty-print flag in the serde_lexpr::print::Options.  Is there a pretty print example someplace?

**Comment by rotty** [comment]
Pretty printing is currently a missing feature. If I understand correctly, you want to use S-expressions for configuration, but the user does not write the config file by herself?

**Comment by bburdette** [comment]
Yes, for generating a default config file, or generating a file from rust according to user inputs.  Thanks for clarifying, from just browsing the code it seemed like there might be some infrastructure there for pretty printing already.
